src/backend/src/routes/users.routes.ts

```diff
@@ -12,7 +13,12 @@ const userRouter = express.Router();
 userRouter.get('/', getAllUsers);
 userRouter.get('/:userId', getSingleUser);
 userRouter.get('/:userId/settings', getUserSettings);
-userRouter.post('/:userId/settings', updateUserSettings);
+userRouter.post(
+  '/:userId/settings',
+  body('defaultTheme').isBoolean().not().isEmpty(),
```

Comment:
It looks like `defaultTheme` should be of type `ThemeName` (an enum), so this check should not be `isIn()` instead of `isBoolean()`. If we use `isIn()`, then we dont need `.not().isEmpty(),` either.